stf.build/include/top.xml

```diff
@@ -408,21 +367,21 @@ limitations under the License.
 	    </or>
 	</condition>
 
-	<macrodef name="download-file" description="Use ${download-tool} to download a file">
+	<macrodef name="download-file" description="Use curl to download a file">
 		<attribute name="srcurl" description="URL of file to download"/>
 		<attribute name="destdir" description="Directory in which to place the downloaded file"/>
 		<attribute name="destfile" description="File name of the downloaded file"/>
 		<sequential>
 			<echo message="Executing macro download-file"/>
 			<echo message="File to download: @{srcurl}"/>
 			<echo message="Destination: @{destdir}/@{destfile}"/>
-			<echo message="Download tool: ${download-tool}"/>
-			<fail unless="downloadtool_available" message="Cannot find ${download-tool} on PATH, needed to download @{srcurl}. Either install ${download-tool} or add it to PATH and retry.  If you are running the build from the Ant view in Eclipse, add ${download-tool} to your PATH before starting Eclipse."/>
+			<echo message="Download tool: curl"/>
+			<fail unless="downloadtool_available" message="Cannot find curl on PATH, needed to download @{srcurl}. Either install curl or add it to PATH and retry.  If you are running the build from the Ant view in Eclipse, add curl to your PATH before starting Eclipse."/>
```

Comment:
Please remove this check <fail unless="downloadtool_available..>. . Since we have gotten rid of downloadtool_available conditional property and we no longer want to check for the availability of download tool (curl for the new case) in this script any more.